packages/core/src/PartialExecutionUtils/__tests__/recreateNodeExecutionStack.test.ts

```diff
@@ -105,8 +107,8 @@ describe('recreateNodeExecutionStack', () => {
 			},
 		]);
 
-		expect(waitingExecution).toEqual({ node: { '0': { main: [null] } } });
-		expect(waitingExecutionSource).toEqual({ node: { '0': { main: [null] } } });
+		expect(waitingExecution).toEqual({});
+		expect(waitingExecutionSource).toEqual({});
```

Comment:
We don't have to add node to the waiting stack. The execution engine will do that after execution trigger.